test/functional/activation_key/info_test.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/associating_commands'
+
+describe 'activation-key info' do
+  before do
+    @cmd = %w(activation-key info)
+  end
+
+  it "includes katello attributes" do
+    params = ['--id=1']
+    ex = api_expects(:activation_keys, :show, "Activation key") do |par|
+      par['id'] == 1
+    end
+
+    json_file = File.join(File.dirname(__FILE__), 'data', 'activation_key.json')
+    ex.returns(JSON.parse(File.read(json_file)))
+
+    result = run_cmd(@cmd + params)
+    # rubocop:disable Style/WordArray
```

Comment:
This is not required and is causing the build to fail.

Reply:
alright. I have removed it.